plugins/haproxy/haproxy.go

```diff
@@ -467,15 +470,17 @@ func (p HaproxyPlugin) GenerateProxyConfig() (*ProxyConfig, error) {
 			}
 		}
 
-	 	container_name := cInfo.Name[1:]
+		container_name := cInfo.Name[1:]
 		up := &Upstream{
 			Addr:          addr,
-                        Container:     container_name,
+			Container:     container_name,
```

Comment:
I don't think this is from this PR but just noting here -- this should be `containerName`.